pkg/controller/plan/kubevirt.go

```diff
@@ -636,11 +636,51 @@ func (r *KubeVirt) createVolumesForOvirt(vm ref.Ref) (err error) {
 		if err != nil {
 			return
 		}
+		err = r.createPodToBindPVC(vm, pvc.Name)
 	}
 
 	return
 }
 
+// Creates a pod associated with PVC to create node bind (wait for consumer)
+func (r *KubeVirt) createPodToBindPVC(vm *plan.VMStatus, pvcName string) error {
+	name := "init"
+	volumes := []core.Volume{}
+	volumes = append(volumes, core.Volume{
+		Name: name,
+		VolumeSource: core.VolumeSource{
+			PersistentVolumeClaim: &core.PersistentVolumeClaimVolumeSource{
+				ClaimName: pvcName,
+			},
+		},
+	})
+	pod := &core.Pod{
+		ObjectMeta: meta.ObjectMeta{
+			Namespace:    r.Plan.Spec.TargetNamespace,
+			Labels:       r.vmLabels(vm.Ref),
+			GenerateName: r.getGeneratedName(vm) + "pvcinit",
+		},
+		Spec: core.PodSpec{
+			RestartPolicy: core.RestartPolicyNever,
+			Containers: []core.Container{
+				{
+					Name:    name,
+					Image:   "ubi8",
+					Command: []string{"/bin/sh"},
+				},
+			},
```

Comment:
it's great that it works without mounting the volumes - that way we can be sure that we can 'populate' even before the execution of this pod stops